apps/docs/src/app/core/component-docs/feed-list-item/examples/fli-mobile/fli-mobile-example.component.html

```diff
@@ -0,0 +1,39 @@
+<div style="max-width: 350px; position: relative">
+    <fd-feed-list [mobile]="true" aria-label="Feed List Item with mobile view">
+        <fd-feed-list-item
+            authorTitle="John Doe"
+            authorLink="http://example.com/"
+            text="Long text, you can toggle view with more/less button. Lorem Ipsum is simply dummy text of the printing and typesetting industry."
+            maxRows="2"
+            ariaLabel="Feed list item example">
+            <fd-feed-list-avatar>
+                <div class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list--link">
```

Comment:
I think `<fd-avatar>` should be used here.

Reply:
@salarenko I updated, but it means that you can use your custom structure and it will place into avatar's box (holder has 
dimensions for preventing extra large content).

Reply:
I'm not sure if I understand. You mean the user can put anything inside `fd-feed-list-avatar` holder and I think thats good. Does using `fd-avatar` instead of `<div class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list--link">` changes something?